st2common/st2common/rbac/syncer.py

```diff
@@ -206,6 +211,28 @@ def sync_users_role_assignments(self, role_assignment_apis):
         LOG.info('User role assignments synchronized')
         return results
 
+    def sync_group_to_role_maps(self, group_to_role_map_apis):
+        LOG.info('Synchronizing group to role maps...')
+
+        # Retrieve all the mappings currently in the db
+        group_to_role_map_dbs = rbac_services.get_all_group_to_role_maps()
```

Comment:
This is not a paginated call. It might be fine for now but I want to call out there is a potential problem here.